improver_tests/calibration/ensemble_calibration/test_ApplyEMOS.py

```diff
@@ -167,7 +142,8 @@ def test_null_probabilities(self):
 
     def test_bias(self):
         """Test emos coefficients that correct a bias"""
-        self.coefficients.data = [0, 1, 1, 1]
+        for coeff, cube in zip([0, 1, 1, 1], self.coefficients):
+            cube.data = coeff
```

Comment:
You can do this much more clearly / neatly by only updating the cube that you need to.  Eg:
```
# update the test "alpha" value
self.coefficients[2].data = 1
```
Or more nicely using cube extraction by name (which avoids the need for a comment), but I'm not sure if you can assign to the source doing that - you'd have to experiment.

Reply:
I've updated this to use the slicing as you suggest. I don't think that this can be done so concisely using a cube extraction approach.